portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -61,6 +63,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         final Long postDate = System.currentTimeMillis();
 
         try {
+            if(!this.isAuthenticated()) {
+                throw new Exception();
+            }
```

Comment:
There is a google cloud documentation that describes how to do it on larger scale https://cloud.google.com/appengine/docs/standard/java/config/webxml#security-auth

Reply:
I think normally you need to create a filter that will have a global effect on the application. However, in our case, the tutorial instructs to implement this in a similar way I did (ie. to invoke the UserService when it's needed). Is it ok if we keep it like this for now?

Reply:
SGTM